tests/kafkatest/services/security/security_config.py

```diff
@@ -166,10 +167,22 @@ def has_sasl_kerberos(self):
 
     @property
     def kafka_opts(self):
+        if SecurityConfig.ZK_JAAS_LOGIN:
+            return "\"-Djava.security.auth.login.config=%s\"" % SecurityConfig.ZK_JAAS_LOGIN
```

Comment:
I don't understand why we do this instead of improving the `has_sasl` branch to be more general. I mean, I understand why we would do this to get some quick results, but it seems to me that we'd want to improve this before merging. Thoughts?